templates/organize/event_funding.html

```diff
@@ -0,0 +1,44 @@
+{% extends "global/base.html" %}
+
+{% block content %}
+<div class="container">
+    <h2>Apply for Sponsorship from our Global Partners</h2>
```

Comment:
I suspect this should have `{% translate` tags in it so that we can provide it in the available languages?